3.3.0 Fails to install

The latest version, 3.3.0, fails on install, at least on linux based systems.  I am seeing the following output on npm install both on my local development environment (ubuntu trusty64 build) and in our travis environment.  Version 3.2.0, which was the most recent version until earlier this morning, still installs just fine.

```
npm install rejoice@3.3.0
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.14
npm WARN e2core@1.0.0 No repository field.
npm ERR! Linux 3.13.0-87-generic
npm ERR! argv "/home/vagrant/.nvm/versions/node/v5.11.1/bin/node" "/home/vagrant/.nvm/versions/node/v5.11.1/bin/npm" "install" "rejoice@3.3.0"
npm ERR! node v5.11.1
npm ERR! npm  v3.8.6
npm ERR! path /vagrant/node_modules/rejoice/bin/rejoice
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/vagrant/node_modules/rejoice/bin/rejoice'
npm ERR! enoent ENOENT: no such file or directory, chmod '/vagrant/node_modules/rejoice/bin/rejoice'
```

--- a/.npmignore
+++ b/.npmignore
@@ -1,3 +1,4 @@
 *
+!bin/**
 !lib/**
 !.npmignore

